src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -161,8 +158,25 @@ const PastAnalysisTable = () => {
       }
       setLoading(false);
     })();
-  }, [tableStates.maxFeedId, tableStates.page, perpage, dispatch, history, stagingDcmImages]);
-
+  }, [tableStates, perpage, stagingDcmImages, dispatch]);
+
+
+  // Polls ChRIS backend and refreshes table if any of the plugins with the given IDs have a terminated status
+  useInterval(async () => {
+    if (tableStates.processingPluginIds) {
+      for (const id of tableStates.processingPluginIds) {
+        const refresh: boolean = await ChrisIntegration.checkIfPluginTerminated(id);
+        if (refresh) {
+          dispatch({
+            type: AnalysisTypes.Update_are_new_imgs_available,
+            payload: { isAvailable: true }
+          });
+          return;
+        }
+      }
+    }
+  }, tableStates.processingPluginIds.length ? 10000 : 0); // Pauses polling if there are no processing rows
```

Comment:
Please move the polling period to app configs so we can easily configure it as needed.

Reply:
Done: https://github.com/darwinai/covidnet_ui/pull/35/commits/43c6a98bdd12882e7f748e84f4ef0c5e4ec29840